cmd/influxd/config.go

```diff
@@ -36,11 +37,21 @@ const (
 
 // Config represents the configuration format for the influxd binary.
 type (
+	Collectd struct {
+		Addr string `toml:"address"`
+		Port uint16 `toml:"port"`
+
+		Database string `toml:"database"`
+		Enabled  bool   `toml:"enabled"`
+		TypesDB  string `toml:"typesdb"`
+	}
```

Comment:
I removed the `type` block when I merged the `join` PR. Can you move this type out of the `type` block as well?